README.md

```diff
@@ -115,6 +115,13 @@ If you're just starting to look at .NET 5.0 and would like to understand more ab
 2. [.NET Core porting documentation](https://docs.microsoft.com/dotnet/core/porting/)
 3. [Documentation of features not available on .NET Core](https://docs.microsoft.com/dotnet/core/porting/net-framework-tech-unavailable)
 
+### Authenticated NuGet sources
+
+As part of upgrading a project, Upgrade Assistant will need to both restore the project's NuGet packages and query configured NuGet sources for information on updated packages. If a project depends on authenticated NuGet sources, some extra steps may be necessary for Upgrade Assistant to work correctly:
+
+1. Upgrade Assistant requires that a v2 .NET Core-compatible NuGet credential provider be installed on the computer if authenticated NuGet sources are used. For example, if you are using authenticated Azure DevOps NuGet feeds, follow [these instructions](https://github.com/microsoft/artifacts-credprovider#setup) to setup a compatible credential provider.
```

Comment:
is there a command I can use to validate I have a v2 credential provider?

Reply:
I've added more details.